sql/src/test/java/org/opensearch/sql/sql/domain/SQLQueryRequestTest.java

```diff
@@ -6,38 +6,43 @@
 
 package org.opensearch.sql.sql.domain;
 
+import static org.junit.jupiter.api.Assertions.assertAll;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 import com.google.common.collect.ImmutableMap;
+import java.util.HashMap;
 import java.util.Map;
 import org.json.JSONObject;
-import org.junit.jupiter.api.Assumptions;
-import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.DisplayNameGeneration;
+import org.junit.jupiter.api.DisplayNameGenerator;
 import org.junit.jupiter.api.Test;
 import org.opensearch.sql.protocol.response.format.Format;
 
+@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
 public class SQLQueryRequestTest {
 
   @Test
-  public void shouldSupportQuery() {
+  public void should_support_query() {
     SQLQueryRequest request = SQLQueryRequestBuilder.request("SELECT 1").build();
     assertTrue(request.isSupported());
   }
 
   @Test
-  public void shouldSupportQueryWithJDBCFormat() {
+  public void should_support_query_with_JDBC_format() {
     SQLQueryRequest request = SQLQueryRequestBuilder.request("SELECT 1")
                                                     .format("jdbc")
                                                     .build();
-    assertTrue(request.isSupported());
-    assertEquals(request.format(), Format.JDBC);
+    assertAll(
+        () -> assertTrue(request.isSupported()),
```

Comment:
Why change to this? Are they not the same?

Reply:
The same, but `assertAll` always runs all assertions, regardless of failures.